### ISSUE TITLE
Wrust crashed v1.5.1

### ISSUE THREAD

**Issue by TheFlyingWaffle** [Wurst-Imperium/Wurst-Client#93]
```
-- Head --
Stacktrace:
	at java.lang.NumberFormatException.forInputString(Unknown Source)
	at java.lang.Integer.parseInt(Unknown Source)
	at java.lang.Integer.valueOf(Unknown Source)
	at tk.wurst_client.files.FileManager.loadBuildings(FileManager.java:803)
	at tk.wurst_client.files.FileManager.init(FileManager.java:100)
	at tk.wurst_client.Client.startClient(Client.java:54)
	at net.minecraft.client.Minecraft.startGame(Minecraft.java:615)

-- Initialization --
Details:
Stacktrace:
	at net.minecraft.client.Minecraft.run(Minecraft.java:428)
	at net.minecraft.client.main.Main.main(Main.java:127)

-- System Details --
Details:
	Minecraft Version: 1.8
	Wurst Client Version: 1.5.1 (latest: unknown)
	Operating System: Windows 7 (x86) version 6.1
	Java Version: 1.8.0_31, Oracle Corporation
	Java VM Version: Java HotSpot(TM) Client VM (mixed mode), Oracle Corporation
	Memory: 124375336 bytes (118 MB) / 242872320 bytes (231 MB) up to 523501568 bytes (499 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx512M -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	Launched Version: Wurst
	LWJGL: 2.9.1
	OpenGL: Intel(R) HD Graphics GL version 2.1.0 - Build 8.15.10.2342, Intel
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because ARB_framebuffer_object is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: No
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	Resource Packs: []
	Current Language: English (US)
	Profiler Position: N/A (disabled)
#@!@# Game crashed! Crash report saved to: #@!@# C:\Users\PRO\AppData\Roaming\.minecraft\crash-reports\crash-2558-02-27_15.47.53-client.txt
```
 
**Im using cracked launcher** I tried another versions and they crashed too.

**Comment by wuofwhat** [comment]
I have this problem too (normal)

**Comment by Alexander01998** [comment]
Should be fixed in Wurst v1.6.

> Im using cracked launcher

Wurst doesn't officially support any cracked launchers.
